Doc/make.bat

```diff
@@ -41,7 +41,7 @@ if exist "%HTMLHELP%" goto :skiphhcsearch
 
 rem Search for HHC in likely places
 set HTMLHELP=
-where hhc /q && set HTMLHELP=hhc && goto :skiphhcsearch
+where hhc /q && set HTMLHELP=hhc&& goto :skiphhcsearch
```

Comment:
For readability, the whitespace problem can be avoided with double quotes around the assignment expression, e.g.  `set "HTMLHEMP=hhc"`.

Reply:
Very good point. Updated.